ocrd_utils/ocrd_utils/constants.py

```diff
@@ -81,6 +83,9 @@
 # Prefix to denote query is regular expression not fixed string
 REGEX_PREFIX = '//'
 
+# Regex for valid mets:file/@ID
+REGEX_FILE_ID = regex_compile('^[A-Za-z][^:]*$')
```

Comment:
That would include spaces and arbitrary Unicode codepoints after the first letter.

[Here](https://stackoverflow.com/a/12332162) is a proposal for a general xs:ID regex. (Probably better to not be over-restrictive either, who knows what we will use that constant for in the future.)

```suggestion
REGEX_FILE_ID = regex_compile(r'^[a-zA-Z_][\w.-]*$')
```